statuslistener/statuslistener_test.go

```diff
@@ -320,19 +321,19 @@ func TestConsumeStatusMessage(t *testing.T) {
 	logging.Log = &log
 
 	header := kafkaGo.Header{Key: "event_type", Value: []byte("availability_status")}
-	header2 := kafkaGo.Header{Key: "x-rh-identity", Value: []byte("Test identity")}
+	header2 := kafkaGo.Header{Key: "x-rh-identity", Value: []byte("eyJpZGVudGl0eSI6eyJhY2NvdW50X251bWJlciI6IjEyMzQ1IiwidXNlciI6IHsiaXNfb3JnX2FkbWluIjp0cnVlfX0sICJpbnRlcm5hbCI6IHsib3JnX2lkIjogIjAwMDAwMSJ9fQo=")}
```

Comment:
Can we have a comment with the decoded contents of the identity? It's just to avoid having to decode it manually :P. `{"identity":{"account_number":"12345","user": {"is_org_admin":true}}, "internal": {"org_id": "000001"}}`

Reply:
Great idea !

Reply:
done 👍